### PR TITLE
Fix session timeout issue

### PR DESCRIPTION
**Pull request by markvr**
Hi, this fixes the sec/min timeout issue and some bugs in the logging.

### CODE REVIEW

**Comment by emre-aydin** [comment]
verify

File: tomcat85/src/main/java/com/hazelcast/session/HazelcastSessionManager.java [hunk 1]
```diff
@@ -262,11 +263,13 @@ public String updateJvmRouteForSession(String sessionId, String newJvmRoute) thr
 
     @Override
     public void remove(Session session) {
+        log.debug("Removed session: " + session.getId());
```
Comment (mesutcelik) [inline tomcat85/src/main/java/com/hazelcast/session/HazelcastSessionManager.java]: this needs to be after `remove` operation. Additionally, it must be `info`

File: tomcat85/src/main/java/com/hazelcast/session/HazelcastSessionManager.java [hunk 2]
```diff
@@ -262,11 +263,13 @@ public String updateJvmRouteForSession(String sessionId, String newJvmRoute) thr
...
         remove(session.getId());
     }
 
     @Override
     public void remove(Session session, boolean update) {
+        log.debug("Removed session: " + session.getId());
```
Comment (mesutcelik) [inline tomcat85/src/main/java/com/hazelcast/session/HazelcastSessionManager.java]: this needs to be after `remove` operation. Additionally, it must be `info`

File: tomcat85/src/main/java/com/hazelcast/session/HazelcastSessionManager.java [hunk 3]
```diff
@@ -211,6 +210,8 @@ public Session findSession(String id) throws IOException {
             if (hazelcastSession == null) {
                 log.info("No Session found for:" + id);
                 return null;
+            } else {
```
Comment (mesutcelik) [inline tomcat85/src/main/java/com/hazelcast/session/HazelcastSessionManager.java]: no need to have `else` block. Additionally, it must be `info`

File: tomcat85/src/main/java/com/hazelcast/session/HazelcastSessionManager.java [hunk 4]
```diff
@@ -85,7 +85,6 @@ public void startInternal() throws LifecycleException {
             Context ctx = getContext();
             String contextPath = ctx.getServletContext().getContextPath();
             log.info("contextPath:" + contextPath);
-            String mapName;
```
Comment (mesutcelik) [inline tomcat85/src/main/java/com/hazelcast/session/HazelcastSessionManager.java]: good catch!

**Comment by emre-aydin** [comment]
verify

**Comment by emre-aydin** [comment]
Hi @markvr 

Thanks for your contribution. Could you please sign our Contributor Agreement to have the PR merged into Hazelcast repository?
https://hazelcast.atlassian.net/wiki/display/COM/Hazelcast+Contributor+Agreement

**Comment by emre-aydin** [comment]
@markvr Any news on the CLA?

**Comment by emre-aydin** [comment]
@markvr Can you please sign the CLA before we can get this merged? If you can't, I'm afraid I will have to close the PR without merging.

**Comment by markvr** [comment]
Ok sorry I'll get on it - I'll need to find a printer.  Is there no way of doing it online?

**Comment by emre-aydin** [comment]
@markvr you can digitally insert your signature into the document and send it - no need to print it.

**Comment by emre-aydin** [comment]
Merged - thanks for the contribution @markvr.

**Pull request merged**